### DISCUSSION TITLE
About tabsets

### DISCUSSION THREAD

**Discussion by robpetrosino** [quarto-dev/quarto-cli#6608] [Q&A] [answered]
### Description

I am using tabsets in my presentation, but I have a few questions about it: 

1. Is there an option that would allow me to move to the next tabset by clicking on the arrow? Right now, to move to the next tab, it looks like I have to use the moouse, but I was wondering if there is a keyboard shortcut to do that. It would help the presentation to flow more smoothly.
2. Is it possible to convert each slide (including the slides within each panel) into pdf pages?

Thanks!

**Comment by rgaiacs** [comment, marked as answer]
> Is there an option that would allow me to move to the next tabset by clicking on the arrow?

I don't no. Maybe you could use [Incremental Lists](https://quarto.org/docs/presentations/revealjs/#incremental-lists) instead of [Tabsets](https://quarto.org/docs/presentations/revealjs/#tabsets).

> Is it possible to convert each slide (including the slides within each panel) into pdf pages?

From the [documentation](https://quarto.org/docs/presentations/revealjs/#tabsets),

> Note that one significant disadvantage to tabsets is that only the first tab will be visible when printing to PDF.

**Reply by robpetrosino** [reply]
That's what I was afraid of. Not very convenient!

**Reply by mcanouil** [reply]
The thing is Reveal.js uses JavaScript to go from one section to the other my basically hiding previous and showing current on top of all previous.
Reveal.js library would need to know there is a level that is not a section/slide.
This would likely require a non trivial JavaScript script (possibly as a Reveal.js plugin which could be turned into a Quarto extension).

**Reply by robpetrosino** [reply]
It makes sense, thank you for explaining this!
